src/queries/display-value.ts

```diff
@@ -8,33 +9,38 @@ import {
   buildQueries,
 } from './all-utils'
 
-function queryAllByDisplayValue(
+const queryAllByDisplayValue: AllByBoundAttribute = (
   container,
   value,
   {exact = true, collapseWhitespace, trim, normalizer} = {},
-) {
+) => {
   checkContainerType(container)
   const matcher = exact ? matches : fuzzyMatches
   const matchNormalizer = makeNormalizer({collapseWhitespace, trim, normalizer})
-  return Array.from(container.querySelectorAll(`input,textarea,select`)).filter(
-    node => {
-      if (node.tagName === 'SELECT') {
-        const selectedOptions = Array.from(node.options).filter(
-          option => option.selected,
-        )
-        return selectedOptions.some(optionNode =>
-          matcher(getNodeText(optionNode), optionNode, value, matchNormalizer),
-        )
-      } else {
-        return matcher(node.value, node, value, matchNormalizer)
-      }
-    },
-  )
+  return Array.from(
+    container.querySelectorAll<HTMLElement>(`input,textarea,select`),
+  ).filter(node => {
+    if (node.tagName === 'SELECT') {
+      const selectedOptions = Array.from(
+        (node as HTMLSelectElement).options,
```

Comment:
', ' in the end seems redandant

Reply:
I think it's their eslint standard because saw it in other places, also it passed my eslint precommit